### PR TITLE
Merged from Kara's repo and created pipeline for public image

### PR DESCRIPTION
**Pull request by mattcui**
The new pipeline for creating public image looks good ->

http://159.122.236.143:8080/pipelines/bosh_public_stemcell?groups=bosh-softlayer-public-stemcell

### CODE REVIEW

File: ci/tasks/build-stemcell.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+get_ip_from_vagrant_ssh_config() {
+  config=$(vagrant ssh-config remote)
+  echo $(echo "$config" | grep HostName | awk '{print $2}')
+}
+
+build_num=$(cat stemcell-version/number | sed 's/\.0$//;s/\.0$//')
+
+pushd bosh-src
+
+# todo check out correct version of bosh-src for that stemcell
```
Comment (maximilien) [inline ci/tasks/build-stemcell.sh]: Why these comments?

**Comment by maximilien** [comment]
OK @mattcui see one comment to remove comments. Also, please ensure there are no passwords or ssh keys in any files (YML or otherwise). Address, resubmit, and I will merge.

File: ci/tasks/build-stemcell.sh (hunk 1 above)
Comment (mattcui) [inline ci/tasks/build-stemcell.sh]: Removed these comments, which should be left by Kara, we don't see any problem without that part of code.

**Comment by mattcui** [comment]
@maximilien Yes, I ever searched the code before, there should be no credential info in the code. I removed those comments. Please review and merge. Thanks.

**Comment by maximilien** [comment]
Ok cool @mattcui. Merging :+1:

**Pull request merged**